gem/templates/rwanda/core/main.html

```diff
@@ -13,10 +13,10 @@
       </p>
     {% endif %}
 
-    {% if 'ninyampinga.com' in request.get_host %}
+    {% if 'ninyampinga.com' in request.get_host or 'ninyampinga-rwanda.qa-hub.ie.gehosting.org' in request.get_host %}
```

Comment:
do we want to leave this here of add it as a var setting?

Reply:
@sewagodimo Good point, updated to a template variable.